docs/use-cases/wallet.md

```diff
@@ -0,0 +1,140 @@
+
+#### 2.1.3 Make Transfers
+
+Making transfers on the OMG Network has the same behavior as on other networks, blockchains, or payment systems. Currently, the network supports only payment transactions, more advanced types will be available in the future. This operation will occupy the majority of data load, thus should be treated with more focus. You can implement a standard transfer functionality using [Make a Transfer guide](https://docs.omg.network/network/transfers).
+
+*Implementation details:*
+
+When you send a transaction, you will often use [`account/get_utxos`](https://docs.omg.network/elixir-omg/docs-ui/?urls.primaryName=master%2Finfo_api_specs#/Account/account_get_utxos) endpoint to get information about the latest UTXOs. Current implementation returns only confirmed UTXOs that are not used for the pending transaction. One of the ways to mitigate this limitation is to block/diable your wallet UI to prevent sending other transactions until the current transaction is confirmed. Otherwise, you may encounter `UTXO not found` issue. Alternatively, you can keep track of all of the UTXOs used for your transactions but this will create additional complexity for your application.
+
+![](/img/wallet/01.png)
+
+#### 2.1.4 Make Withdrawals (Exits)
+
+Using OMG Network as a value transfer layer offers a faster and cheaper alternative if your work requires frequent operations on the Ethereum network. Most of the time you will hold a certain amount of assets on the OMG Network, and periodically withdraw your funds back to Ethereum. This is the most complicated technical and UX flow. You can implement exits functionality using [Deposit Funds guide](https://docs.omg.network/network/deposits).
+
+*Implementation details:*
+
+1. The entire exit flow for mobile can be very complicated. It involves dealing with challenges, invalid exits, covering standard and in-flight exits. Thus, if you don't have enough time and resources for implementing such functionality, we recommend not to include it in your wallet at all. Exits are perceived as a less frequent activity on a network and can be accomplished with a Web Wallet provided by the OMG Network team.
+2. There's no endpoint to query exit data in the [Watcher Info API](https://docs.omg.network/elixir-omg/docs-ui/?urls.primaryName=master%2Finfo_api_specs). It is available only in [omg-js](https://github.com/omgnetwork/omg-js) library.
+
+### 2.2 Design Guideline
+
+Later 2 mobile UI and UX can be challenging and confusing, thus our job is to make user flows intuitive, appealing, and easy to navigate. Below you can see recommendations from our team.
```

Comment:
> Later 2 mobile UI and UX can be challenging and confusing,

what do you mean by this?

Reply:
Because this is all new and there are no clear established rules by any community or company how to create an intuitive and easy-to-use design. But that's similar to all blockchain wallets I guess : )

Reply:
Should be "layer 2" not "later 2" ;)

Reply:
that's already done : )